src/features/network-modifications/twoWindingsTransformer/characteristics/TwoWindingsTransformerCharacteristicsPane.tsx

```
@@ -0,0 +1,136 @@
+        <FloatInput
+            name={`${id}.${FieldConstants.RATED_U2}`}
+            label="RatedVoltage"
+            adornment={VoltageAdornment}
+            previousValue={twtToModify?.ratedU2}
+            clearable={isModification}
+        />
+    );
+
+    return (
+        <>
+            <GridSection title={'Characteristics'} />
```

ESLint

[error] Curly braces are unnecessary here.

(react/jsx-curly-brace-presence)